src/transformers/Lichess.js

```diff
@@ -0,0 +1,19 @@
+const { URL } = require('url');
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    ['lichess.org', 'www.lichess.org'].includes(host) &&
+    !pathname.includes('/embed/') &&
```

Comment:
You should copy over the `includesSomeOfArray` function from the Spotify transformer and use that one instead here

Reply:
Hi, I have updated, it on local repo, and checked the tests.  I would put it on the remote repo shortly.